### PR TITLE
DEU-subs-Lucien-Ducastel-01

### PR DESCRIPTION
**Pull request by therrndobler**
This first clip is ready for review. 
I tried to base myself more on the French text. So I added/changed a few small things, e.g. line 50. 
Line 142 seemed to differ in French and English. Let me know if you’d rather stick to the English text here.

### CODE REVIEW

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 1]
```diff
@@ -122,22 +122,22 @@ de:
...
-          text: The strikes of 1936 were the beginning of it all.
+          text: Mit den Streiks 1936 fing alles an.
         sequence-23:
           s: 23
           from: 00:02:25,00
           to: 00:02:33,27
-          text: There was a labour movement that manifested itself. I was already
-            part of the working class.
+          text: Daraufhin bildete sich die Arbeiterbeweung heraus. Zu dieser Zeit war ich
+          bereits Teil des Arbeiterumfelds.
```
Comment (pixelpogo) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: For me the term `Arbeiterumfelds` doesn't sound correct. The English translation `working class` is also not perfect in my opinion. How about sticking to the French and use milieux like `Arbeitermilieus`?

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 2]
```diff
@@ -122,22 +122,22 @@ de:
...
         sequence-24:
           s: 24
           from: 00:02:33,27
           to: 00:02:42,26
-          text: In these conditions the 1936 movement manifested itself a little bit
-            everywhere and there were big demonstrations.
+          text: Unter diesen Umständen stieg die Bewegung von 1936 überall
```
Comment (pixelpogo) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: My suggestion: `Unter diesen Umständen wurde die Bewegung von 1936 überall ein wenig stärker und es gab diese große Demonstration von 1936`

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 3]
```diff
@@ -122,22 +122,22 @@ de:
...
+           ein wenig weiter auf und es gab große Demonstrationen.
         sequence-25:
           s: 25
           from: 00:02:42,26
           to: 00:02:48,12
-          text: It meant very much to the working class movement in France. It was
-            important.
+          text: Alles, was das für die Arbeiterbewegung in Frankreich bedeutete.
```
Comment (pixelpogo) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: and continuing: `und was das alles für die Arbeiterbewegung in Frankreich bedeutete, war sehr wichtig. Das ist gewiss.`

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 4]
```diff
@@ -19,13 +19,13 @@ de:
           s: 3
           from: 00:00:12,25
           to: 00:00:25,18
-          text: For three years, I was an apprentice butcher, but did not like it
-            at all. I started working in construction, as a metal worker.
+          text: Ich war drei Jahre lang Metzgerlehrling, aber das gefiel mir gar
+            nicht. Dann fing ich an, auf dem Bau zu arbeiten, im Metallbau.
```
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: The comma before "aber" is mandatory. 
I'm a fan of commas :-D so I generally tend to put a comma before an infinitive with "zu". But that is optional, you're right. So we can delete the comma after "an" if you prefer it :)

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 5]
```diff
@@ -35,41 +35,41 @@ de:
           s: 6
           from: 00:00:35,20
           to: 00:00:42,19
-          text: My parents, my father was a metal worker, a hard working man. He repaired
-            ships.
+          text: Meine Eltern … mein Vater war ein Handwerker, ein Metallarbeiter,
```
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: I'm afraid as opposed to English, in German there needs to be a space before an ellipsis. Unless the word itself is cut off: So "Meine Eltern …" vs. "Mein Elt…" ;)

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 6]
```diff
@@ -35,41 +35,41 @@ de:
...
           to: 00:00:53,12
-          text: ''
+          text: Sie arbeiteten beide und ich ging zur Schule. Es gab keine Probleme.
         sequence-9:
           s: 9
           from: 00:00:53,12
           to: 00:01:01,06
-          text: I have my elementary school certificate. I was barely twelve years
-            old.
+          text: Ich hatte mein Abschlusszeugnis von der Grundschule …
```
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: same as above

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 7]
```diff
@@ -35,41 +35,41 @@ de:
...
           from: 00:01:01,06
           to: 00:01:09,05
-          text: My birthday is in the end of August and I got the certificate in June.
+          text: Mein Geburtstag ist Ende August und ich bekam mein Zeugnis im Juni.
         sequence-11:
           s: 11
           from: 00:01:09,05
           to: 00:01:16,12
-          text: It was important to me. Afterwards, I started working.
+          text: Das war mir sehr wichtig. Danach fing ich an, zu arbeiten.
```
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: yes, we can delete that

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 8]
```diff
@@ -106,13 +106,13 @@ de:
...
-          text: I was 16 years old in 1936 and I worked in factories.
+          text: 1936 war ich 16 Jahre alt und ich fing an, in Fabriken zu arbeiten.
         sequence-20:
           s: 20
           from: 00:01:56,22
           to: 00:02:09,21
-          text: I had simply become a worker and got to know the whole period of 1936,
-            the strikes and all that it represented.
+          text: Ich war einfach ein Handwerker geworden und ich durchlebte den ganzen
+            Zeitraum 1936, die Streiks 1936 und alles, was das bedeutete …
```
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: same as above

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml (hunk 1 above)
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: "Arbeitermilieus" find ich gut!

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 9]
```diff
@@ -90,13 +90,13 @@ de:
           s: 16
           from: 00:01:34,04
           to: 00:01:41,26
-          text: Then I started working in construction, in civil projects, doing repair
-            work on ships.
+          text: Dann fing ich an, im Hoch- und Tiefbau zu arbeiten,
```
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: same as above

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 10]
```diff
@@ -106,13 +106,13 @@ de:
           s: 19
           from: 00:01:48,03
           to: 00:01:56,22
-          text: I was 16 years old in 1936 and I worked in factories.
+          text: 1936 war ich 16 Jahre alt und ich fing an, in Fabriken zu arbeiten.
```
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: same as above

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml (hunk 2 above)
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: Agree! I'd add a full stop at the end here.

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml (hunk 3 above)
Comment (therrndobler) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: Start a new sentence with "Und …"

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml (hunk 4 above)
Comment (pixelpogo) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: I'm in favor of as less commas as possible. :-)

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml (hunk 5 above)
Comment (pixelpogo) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: Ooops, you are right. 👍

File: locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml [hunk 11]
```diff
@@ -35,41 +35,41 @@ de:
...
-          text: I have my elementary school certificate. I was barely twelve years
-            old.
+          text: Ich hatte mein Abschlusszeugnis von der Grundschule …
+            Ich war kaum zwölf Jahre alt.
         sequence-10:
           s: 10
           from: 00:01:01,06
           to: 00:01:09,05
-          text: My birthday is in the end of August and I got the certificate in June.
+          text: Mein Geburtstag ist Ende August und ich bekam mein Zeugnis im Juni.
```
Comment (pixelpogo) [inline locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml]: This is a very good translation! :-)

**Comment by pixelpogo** [comment]
Cool! Looks good. 👍

**Pull request merged**